sotopia/generation_utils/generate.py

```diff
@@ -501,7 +501,15 @@ async def agenerate(
         instantiated_prompt = prompt_result.messages[0].content
         assert isinstance(output_parser, PydanticOutputParser)
         assert isinstance(instantiated_prompt, str)
-        client = OpenAI()
+        if model_name.startswith("custom"):
+            client = OpenAI(
```

Comment:
Can we make sure telling people that the custom model has to go through the openAI SDK?

Reply:
Done, added a doc. Closes #215